README.md

```diff
@@ -102,6 +102,9 @@ Instructions for developers for testing [here](docs/TESTING.md). If you want to
 
 ### Support JoinMarket and bitcoin privacy
 
-Donate to help make JoinMarket even better: `bc1q5x02zqj5nshw0yhx2s4tj75z6vkvuvww26jak5` or `1AZgQZWYRteh6UyF87hwuvyWj73NvWKpL`. Signed bitcoin addresses can be found [here](docs/signed-donation-address.txt).
+Donate to help make JoinMarket even better: [https://bitcoinprivacy.me/joinmarket-donations](Obtain a bitcoin address here)
```

Comment:
Should be should `[text](URL)`, not `[URL](text)` :)

Reply:
derp. Thanks.